test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -878,6 +1063,20 @@ private enum EmptyEnum
         {
         }
 
+        private enum EnumWithLocalizedDisplayNames
```

Comment:
They were leftovers from a previous iteration, cleaned now.

Filed #5217 for Group.